warp10/src/main/sh/package.sh

```diff
@@ -15,73 +15,104 @@
-# Copy startup scripts
-sed -e "s/@VERSION@/${VERSION}/g" ../src/main/sh/warp10.service >> ${WARP10_HOME}/bin/warp10.service
-sed -e "s/@VERSION@/${VERSION}/g" ../src/main/sh/warp10.init >> ${WARP10_HOME}/bin/warp10.init
-sed -e "s/@VERSION@/${VERSION}/g" ../src/main/sh/snapshot.sh >> ${WARP10_HOME}/bin/snapshot.sh
-sed -e "s/@VERSION@/${VERSION}/g" ../src/main/sh/warp10.sh >> ${WARP10_HOME}/bin/warp10.sh
-sed -e "s/@VERSION@/${VERSION}/g" ../src/main/sh/warp10-env.sh >> ${WARP10_HOME}/bin/warp10-env.sh
+##
+## Copy startup scripts
+##
+cp ../src/main/sh/warp10.service "${WARP10_HOME}/bin/warp10.service"
+sed -e "s/@VERSION@/${VERSION}/g" ../src/main/sh/warp10.sh >"${WARP10_HOME}/bin/warp10.sh"
 
-# Copy log4j README, config, runner, bootstrap...
-cp ../../etc/bootstrap/*.mc2 ${WARP10_HOME}/etc/bootstrap
-cp ../../etc/install/README.md ${WARP10_HOME}
-cp ../../CHANGELOG.md ${WARP10_HOME}
-cp ../../etc/runners/*.mc2* ${WARP10_HOME}/runners/test/60000
-cp ../../etc/calls/*.sh ${WARP10_HOME}/calls
-cp ../../etc/calls/*.py ${WARP10_HOME}/calls
-cp ../../etc/macros/* ${WARP10_HOME}/macros
-sed -e "s/@VERSION@/${VERSION}/g" ../../etc/log4j.properties >> ${WARP10_HOME}/etc/log4j.properties
+##
+## Copy log4j, README, config, runner, bootstrap...
+##
+cp ../../etc/bootstrap/egress.mc2 "${WARP10_HOME}/etc/bootstrap/egress.mc2"
+cp ../../etc/bootstrap/mobius.mc2 "${WARP10_HOME}/etc/bootstrap/mobius.mc2"
+cp ../../etc/bootstrap/runner.mc2 "${WARP10_HOME}/etc/bootstrap/runner.mc2"
+cp ../../etc/install/README.md "${WARP10_HOME}/README.md"
+cp ../../CHANGELOG.md "${WARP10_HOME}/CHANGELOG.md"
+cp ../../etc/runners/runner.mc2.DISABLE "${WARP10_HOME}/runners/test/60000/runner.mc2.DISABLE"
+cp ../../etc/calls/testcall.sh "${WARP10_HOME}/calls/testcall.sh"
+cp ../../etc/calls/testcall.py "${WARP10_HOME}/calls/testcall.py"
+cp ../../etc/macros/README "${WARP10_HOME}/macros/README"
+sed -e "s/@VERSION@/${VERSION}/g" ../../etc/log4j.properties >"${WARP10_HOME}/etc/log4j.properties"
+cp ../../etc/warp10-env.sh "${WARP10_HOME}/etc/warp10-env.sh"
 
-# Copy template configuration
-cp -r ../../etc/conf.templates ${WARP10_HOME}/
-sed -i -e "s/@VERSION@/${VERSION}/g" ${WARP10_HOME}/conf.templates/*/*
-cp  ../../etc/warp10-tokengen.mc2 ${WARP10_HOME}/templates/warp10-tokengen.mc2
+##
+## Copy template configuration
+##
+cp -r ../../etc/conf.templates "${WARP10_HOME}"
+sed -i -e "s/@VERSION@/${VERSION}/g" "${WARP10_HOME}"/conf.templates/*/*
+cp ../../etc/demo-tokengen.mc2 "${WARP10_HOME}/tokens/demo-tokengen.mc2"
 
 # Copy jars
-cp ../build/libs/warp10-${VERSION}.jar ${WARP10_HOME}/bin/warp10-${VERSION}.jar
+cp "../build/libs/warp10-${VERSION}.jar" "${WARP10_HOME}/bin/warp10-${VERSION}.jar"
 
-find ${WARP10_HOME} -type d -exec chmod 755 {} \;
-find ${WARP10_HOME} -type f -exec chmod 644 {} \;
-find ${WARP10_HOME} -type f \( -name "*.sh" -o -name "*.py" -o -name "*.init" \) -exec chmod 755 {} \;
+##
+## Fix permissions
+##
+find "${WARP10_HOME}" -type d -exec chmod 755 {} \;
```

Comment:
etc should be in 700 given it hase secrets. leveldb should also be in 700 since it contains the data